### ISSUE TITLE
Uncaught TypeError: n.props.saveVideo is not a function

### ISSUE THREAD

**Issue by Shaxadhere** [Prakshal-Jain/React-Video-Editor#8]
When I click on save, this error appears in the console. I guess there's some issue with saving.

**Comment by Prakshal-Jain** [comment]
Yes. We are working on it and this will be fixed soon. Thank you for reporting this 🙂

**Comment by Acarvajal904** [comment]
any update on this issue ? @Prakshal-Jain

**Comment by Prakshal-Jain** [comment]
@Acarvajal904 Thank you for reaching out. We have discontinued working on the old version of the Video Editor.
The **good news** is, I am working on second version (V2) of the editor, which is far better, have way many features, more efficient and optimized than the current one.

If you wish to try it, you can find it here: https://github.com/Prakshal-Jain/Reactive/tree/v2

Feel free to provide any suggestions and let me know if you want to see any feature in V2. I will be happy to work collaboratively on that 🙂
As I am actively working on v2, it will be launched by mid-July.
Thank you!
